src/index.js

```diff
@@ -59,20 +60,29 @@ const initialize = ({
 
 	let cachedState;
 
+	const ctsApiVersion = 'v1';
 	const services = {};
 	const ctsSearch = () => {
 		const service = ClinicalTrialsServiceFactory.create(
 			ctsHostname,
-			'v1',
+			ctsApiVersion,
 			ctsProtocol,
 			ctsPort
 		);
 		return service;
 	};
 	services.ctsSearch = ctsSearch;
+	const ctsApiEndpointV1 = `${ctsProtocol}://${ctsHostname}${
```

Comment:
I thought we were going to go with just creating initialization props `ctsApiEndpointV1` and `ctsApiEndpointV2` so that we didn't need to go through this building process

Reply:
Looks like I misspoke.  Leave the v1 stuff in and we'll pass in v2 endpoint in initialization.